src/app/views/interactives/InteractivesIndex.test.js

```diff
@@ -142,21 +143,144 @@ describe("Interactives index", () => {
+                                },
+                                metadata: {
+                                    title: "Title 1",
+                                    label: "Label 1",
+                                    internal_id: "internal_id_1",
+                                    slug: "label-1",
+                                },
+                                published: false,
+                                state: "ArchiveUploaded",
+                            },
+                            {
+                                id: "65a93ed2-31a1-4bd5-89dd-9d44b8cda05c",
+                                archive: {
+                                    name: "test2.zip",
+                                },
+                                metadata: {
+                                    title: "Title 2",
+                                    label: "Label 2",
+                                    internal_id: "internal_id_2",
+                                    slug: "label-2",
+                                },
+                                published: true,
+                                state: "ArchiveUploaded",
+                            },
+                            {
+                                id: "65a93ed2-31a1-4bd5-89dd-9d44b8rgu05c",
+                                archive: {
+                                    name: "test3.zip",
+                                },
+                                metadata: {
+                                    title: "Title 3",
+                                    label: "Label 3",
+                                    internal_id: "internal_id_3",
+                                    slug: "label-3",
+                                },
+                                published: true,
+                                state: "ArchiveUploaded",
+                            },
+                        ],
+                        count: 1,
+                        offset: 0,
+                        limit: 20,
+                        total_count: 3,
+                    },
+                })
+            );
+
+            const { items: allItems } = await getAll();
+
+            const list = screen.getByRole("list");
+            const { queryAllByRole } = within(list);
+
+            const updatedState = Object.assign({}, initialState, {
+                filteredInteractives: allItems,
+            });
+
+            useSelectorMock.mockReturnValue(updatedState);
+
+            rerender(<InteractivesIndex {...defaultProps} />);
+
+            const newItems = queryAllByRole("listitem");
+            expect(newItems.length).toBe(3);
+
             const applyButton = screen.getByText("Apply");
-            screen.getByLabelText("Internal ID").value = "Query";
+            screen.getByLabelText("Title").value = "Label 1";
             fireEvent.click(applyButton);
             expect(filterInteractives).toHaveBeenCalled();
```

Comment:
I meant to expect(filterInteractives).toHaveBeenCalledWith({...... });

Reply:
addressed